.github/workflows/release-nmtc.yml

```diff
@@ -0,0 +1,39 @@
+name: Release nmtc
+
+on:
```

Comment:
Let's create releases as it is intended.

Reply:
Let's use [workflow_dispatch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch). [Blog](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/).

```yaml
on:
  workflow_dispatch
```

Special GUI on website should appear.